lib/node_modules/@stdlib/blas/base/drot/lib/drot.js

```diff
@@ -0,0 +1,87 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MAIN //
+
+/**
+* Applies a plane rotation.
+*
+* @param {PositiveInteger} N - number of indexed elements
+* @param {Float64Array} x - first input array
+* @param {integer} strideX - `x` stride length
+* @param {Float64Array} y - second input array
+* @param {integer} strideY - `y` stride length
+* @param {number} c - cosine of the angle of rotation
+* @param {number} s - sine of the angle of rotation
+* @returns {Float64Array} `y`
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var x = new Float64Array( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+* var y = new Float64Array( [ 6.0, 7.0, 8.0, 9.0, 10.0 ] );
+*
+* drot( x.length, x, 1, y, 1, 0.707, 0.707 );
+* // x => <Float64Array>[ 4.949, ~6.363, ~7.777, ~9.191, ~10.605 ]
+* // y => <Float64Array>[ 3.535, 3.535, ~3.535, ~3.535, ~3.535 ]
```

Comment:
Why are the return values in `y` all the same? And some are approx but others are not?

Reply:
I will check this example once.

Reply:
`y[ i ] = c * y[ i ]  - s * x[ i ]`. In this case `c = s = 0.707` thus `y[i] = 0.707 * ( y[i]-x[i] )` also, ` abs(y[i]-x[i]) = 5.0`  for all values hence the absolute answer for `y` is same.

Reply:
@Pranavchiku Thanks. Makes sense. We should still ensure that approx is indicated consistently.

Reply:
In this case, it looks like it should be `3.535` exactly for all values.